wheelsproxy/utils.py

```diff
@@ -88,3 +89,10 @@ def retry_call(times, func, *args, **kwargs):
                 times -= 1
             else:
                 raise
+
+
+def exponential_backoff(retries, max_wait=None, min_wait=0):
+    wait = random.uniform(2, 4) ** retries
```

Comment:
as before, how fast is pypi? maybe the time must be finetuned later

Reply:
It should just be a couple of seconds. From what I was able to gather, they set the cache to 1 day and bust manually.

The exponential backoff goes anywere from 1 seconds up to 85 seconds.